### PR TITLE
modify osdeploy test cases

### PR DESCRIPTION
**Pull request by junxiawang**
@tingtli  please help review osdeploy  test cases

### CODE REVIEW

File: xCAT-test/autotest/testcase/osdeploy/cases0 [hunk 1]
```diff
@@ -68,5 +68,260 @@ check:rc==0
 cmd:rm -rf /tmp/osdeploy.test
 check:rc==0
 end
+start:osdeploy_n_invalidnode
+description: osdeploy -n node invalid
+cmd:test=`lsdef test_node`;if [ $test eq 0 ]; then lsdef -l test_node -z >/tmp/node.standa ;rmdef test_node;fi
```
Comment (tingtli) [inline xCAT-test/autotest/testcase/osdeploy/cases0]: where do you define test_node?

**Comment by tingtli** [comment]
@junxiawang  Here is my comments:
1. it seems -t is not tested in the cases
2. -r should have some complicated value, e.g 1h30m

File: xCAT-test/autotest/testcase/osdeploy/cases0 (hunk 1 above)
Comment (junxiawang) [inline xCAT-test/autotest/testcase/osdeploy/cases0]: just no definition so xcatprobe osdeploy could report error

**Comment by tingtli** [comment]
i agree to merge.

**Pull request merged**